website/docs/plugin/index.mdx

```diff
@@ -18,7 +18,7 @@ Terraform is logically split into two main parts:
 - Learn more about [how Terraform Core interacts with plugins](/plugin/how-terraform-works).
 - Learn the [design principles](/plugin/hashicorp-provider-design-principles) HashiCorp developers follow when creating providers.
 - Decide [which SDK](/plugin/which-sdk) is right for your provider.
-- Try these hands-on tutorials: [Call APIs with Terraform Providers (SDKv2)](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) and [Implement Create and Read (Framework)](https://learn.hashicorp.com/tutorials/terraform/plugin-framework-create?in=terraform/providers)
+- Try these hands-on tutorials: [Call APIs with Terraform Providers (SDKv2)](https://learn.hashicorp.com/collections/terraform/providers?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) and [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-resource-create)
```

Comment:
Two questions:

- Should we adjust the other Learn link to be relative?
- Should we point to the framework collection rather than a specific tutorial?

Maybe something like:

```suggestion
- Try these hands-on tutorials: [Call APIs with Terraform Providers (SDKv2)](/terraform/tutorials/providers) and [Implement a Provider with the Terraform Plugin Framework](/terraform/tutorials/providers-plugin-framework)
```

Reply:
We could (or maybe should?) flip the order of these links as well to potentially encourage framework slightly more over sdk/v2.

Reply:
oooooo i like that! let me do it real quick